packages/react-composites/tests/browser/chat/app/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const Dotenv = require('dotenv-webpack');
+
+module.exports = {
+  entry: path.join(__dirname, 'index.tsx'),
+  devtool: 'eval-source-map',
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js'],
+    alias: {
```

Comment:
hmm, note to self there's a of these alias configs spread across tsconfigs and webpacks files, I wonder if there's a global place we can put them to avoid much redundancy and ensure they're kept consistent